generic_config_updater/patch_sorter.py

```diff
@@ -1014,6 +1015,165 @@ def create(self, algorithm=Algorithm.DFS):
 
         return sorter
 
+class StrictPatchSorter:
+    def __init__(self, config_wrapper, patch_wrapper, inner_patch_sorter=None):
+        self.logger = genericUpdaterLogging.get_logger(title="Patch Sorter - Strict", print_all_to_console=True)
+        self.config_wrapper = config_wrapper
+        self.patch_wrapper = patch_wrapper
+        self.inner_patch_sorter = inner_patch_sorter if inner_patch_sorter else PatchSorter(config_wrapper, patch_wrapper)
+
+    def sort(self, patch, algorithm=Algorithm.DFS):
+        current_config = self.config_wrapper.get_config_db_as_json()
+        target_config = self.patch_wrapper.simulate_patch(patch, current_config)
```

Comment:
>target_config [](http://example.com/codeflow?start=8&length=13)

Move this line after "validate patch is only updating tables with yang models", so it will efficient in cases.